qa/smoke-test-plugins/build.gradle

```diff
@@ -39,3 +41,7 @@ tasks.named("processTestResources").configure {
   inputs.properties(expansions)
   MavenFilteringHack.filter(it, expansions)
 }
+
+if (Os.isFamily(Os.FAMILY_WINDOWS)) {
```

Comment:
Could you add "awaits fix" comments with links to https://github.com/elastic/elasticsearch/issues/73539 ?

Reply:
Thanks. Comments added.